### DISCUSSION TITLE
Had to rollback from 5.3.2.1 to 5.3.2.0

### DISCUSSION THREAD

**Discussion by choskins-godaddy** [bootscore/bootscore#361] [Q&A] [answered]
Ultimately, it came down to this:

<img width="338" alt="image" src="https://user-images.githubusercontent.com/104801366/212808572-0bbacce3-1394-4625-8967-f65154d11611.png">

At the bottom of my checkout page it said, "a critical error has occured" or something like that. I did also manually update my child theme to 5.3.2.1 but that is rolled back too.

**Comment by crftwrk** [comment, marked as answer]
The only error belongs to bootScore is the number one with the `terms.php`, but 5.3.2.1 did not any changes there and child must not be updated. The rest are WP and Woo errors which have nothing to do with the theme, means bugs are somewhere else but not in theme.

However, we can't check this without having a link to the site where this happens with latest 5.3.2.1 main-theme.

**Comment by choskins-godaddy** [comment]
I pushed my live site to my staging site and updated to 5.3.2.1 on the staging site and it's fine -- no error on checkout . I suppose I could sync staging to production but that scares me. What could be different? Why would one site behave differently if it supposedly has the same code and DB? The plugins are the same except for a few that require an active license like Wordfence and AIOSEO. I tried disabling them on production and it didn't fix anything.

If I sync staging to production, I could always restore a backup but I fear something weird would happen.

It would be really interesting to me if the problem still occurs on production after a clone down.  @crftwrk do you know of a way to compare wordpress instances? I can always pull down the code via ftp and do a file compare but I'm not sure how I would compare databases.

**Comment by crftwrk** [comment]
> do you know of a way to compare wordpress instances? I can always pull down the code via ftp and do a file compare but I'm not sure how I would compare databases.

Unfortunately not. If I'm facing a problem like yours, first thing I do is to grab a copy of the site and test it on another server. If that works fine, then I check the server settings or move directly to another one. Maybe ask your hoster?

**Comment by choskins-godaddy** [comment]
Yesterday, I took the plunge and pushed stage to prod and guess what? It worked! So, now I'm up and running on latest!
